compiler/plugins/target/ROCM/builtins/mlir_ukernel/ukernel_patterns_gfx942.mlir

```diff
@@ -314,10 +314,8 @@ pdl.pattern @annotate_matmul_like_f16_medium_expanded : benefit(1) {
   // M, N >= 1024, K >= 256
   %c1024 = pdl.attribute = 1024
 
-  // TODO: Kernel specialization is needed to apply this strategy selectively at
-  // runtime. Additionally model exports don't specify lower bounds so it is
-  // impossible to use this strategy with this check.
-  // pdl.apply_native_constraint "dimIsBound"(%lhs, %c0, %c4, %empty : !pdl.value, !pdl.attribute, !pdl.attribute, !pdl.attribute)
+  // From experimental results, we should start using this ukernel if B * M >= 1024.
```

Comment:
Maybe we should link the result, which I believe is the PR description, to the comment. E.g.,
```suggestion
  // From experimental results, see #22393, we should start using this ukernel if B * M >= 1024.
```

I think it helps future decision for whether we can drop or update the conditions.